### ISSUE TITLE
UJR suddenly refuses to start

### ISSUE THREAD

**Issue by rlangeoi** [evilC/AHK-Universal-Joystick-Remapper#24]
Hi there,
Great soft here that I've been using for quite some time , but unfortunately it suddenly stopped working for no apparent reason.
Every time I try to start UJR I get a window reading 

> Error: The following variable name contains an illegal character: "<html xmlns" 
Line#
---> 2389: out[%k%] := SubStr(A_LoopFiels, p+1)
The current thread will exit.

Then I get the little green icon in my taskbar, but I can't access the config panel. I changed nothing important on my computer or UJR recently and it was working perfectly fine before.

I've tried re-downloading, deleting evrything in the folder except the .exe and it still won't work properly. What can I do ?
Thx in advance

**Comment by evilC** [comment]
Sounds like one of the files got corrupted.  
`<html xmlns` is HTML code, and there is no HTML code in UJR that I can think of.  
Also, `A_LoopFiels` is mis-spelled
Your settings are in UJR.ini, so would suggest backing that up and then replacing all the UJR files with fresh ones (Including the exe).

**Comment by rlangeoi** [comment]
Thanks for the quick reply, I mistyped Field.
I've tried downloading the executable again from this repo, put it in a fresh folder and it still won't work

**Comment by tacosundae** [comment]
I can replicate rlangeoi's issue. Fresh install of windows 10, stick the exe in a folder and execute. missing config file notification as usual, then the strange error message reported above.

**Comment by rlangeoi** [comment]
I should have mentionned yeah, it comes after the missing config file window in the fresh install

**Comment by h55-ud3** [comment]
It will work if you disable network before launching UJR. Something is breaking when it phones home.

**Comment by rlangeoi** [comment]
It does work. Thanks !

**Comment by evilC** [comment]
Yeah, sorry guys, I accidentally toasted my website yesterday. Should be back up at some point today

**Comment by evilC** [comment]
evilc.com is now back up.

**Comment by NiklasMerz** [comment]
Unfortunately this is happening again

**Comment by Blood-Monarch** [comment]
This is happening again, works when disconnecting from internet before running the ujr.exe. The question I have is, why is this software contacting an external site before running, when it obviously doesnt need too!!  ???

**Comment by pastranario3** [comment]
How strange that this application connects to somewhere. A pity that it fails again, it was working perfectly.

**Comment by Blood-Monarch** [comment]
Actually from the 'read me' I can now see why its 'phoning home' 

"WARNING: UJR does "Phone Home" to check version number (it reads a text file from my web site) so it can let you know if a newer version is available. This behavior is non-malicious, but any attempt by any version of UJR to do anything else network-wise should be considered malicious and a result of a modified or infected file."

Perhaps it might be better to have a 'check update' button instead? If the website went off line permanently, it would be a bit of a pain having to crawl under my desk everytime I run the prog to disconnect the internet cable. I appreciate its free software and I'm benefiting from someone else working for free, which i'm very thankful for, and don't like to moan, but if this 'phoning home' could be removed it would be much appreciated

**Comment by Donric** [comment]
no need to crawl anywhere. just block UJR in windows firewall

**Comment by Blood-Monarch** [comment]
I guess I could, but I'm using the updated version UCR now. I didnt know there was a newer version, so not surprising the website for UJR is closed, as its no longer needed.

**Comment by tzanislav** [comment]
I solved in my situation. I **blocked it in the firewall outbound rules** and it works and launches instantly. Because I had an issue that it took 2 min if it worked.
